beat/config.py

```diff
@@ -696,6 +700,53 @@ def init_waveforms(self, wavenames=['any_P']):
         for wavename in wavenames:
             self.waveforms.append(WaveformFitConfig(name=wavename))
 
+##Mahdi
+
+class PolarityGFConfig(NonlinearGFConfig):
+
+    reference_location = ReferenceLocation.T(
+    default=None,
+    help="Reference location for the midpoint of the Green's Function "
+            "grid.",
+    optional=True)
+    sample_rate = Float.T(
```

Comment:
indentation, here rather seismic sample rate like 1Hz is useful, you pasted from the geodetic-i.e. viscoelastic rates ;)

Reply:
Done!